test/common/tests.go

```diff
@@ -33,4 +33,9 @@ var (
 		{"F06 - Verify Replicas Scale correctly in Apicurito", TestReplicasInApicurito},
 		{"F08 - Verify Replicas Scale correctly in RHSSO and user SSO", TestReplicasInRHSSOAndUserSSO},
 	}
+
+	NIGHTLY_TESTS = []TestCase{
```

Comment:
I like the concept of spit these tests form the rest, but I wouldn't call them nightly, as nightly indicate when the test will be executed and not what kind of test is it. I propose to call them `SYSTEM_TESTS` or even `DESTRUCTIVE_TESTS`. To go even further it could make sense to rename the `AFTER_INSTALL_TESTS` to `HAPPY_PATH_TESTS` or something else

Reply:
I think main differentiator of this test is how long it will take to execute it, and that's why we didn't want it to run on each PR. 
But you have a good point about it being a destructive test... 🤔

Reply:
Yeah, I'm good with changing the name of the flag to either or, but yeah the idea was to allow longer / certain tests on specifc pipelines (nightly in this case). Do you guys any preference of  `SYSTEM_TESTS` or `DESTRUCTIVE_TESTS` ?

Reply:
IMHO `DESTRUCTIVE_TESTS` is better than `SYSTEM_TESTS`.

Reply:
Yes `DESTRUCTIVE_TESTS` sounds better, also `LONGRUNNING_TESTS` or `MARATHON_TESTS` if we want to highlight the fact that they take a long time :D

Reply:
Cool, i'll go with `DESTRUCTIVE_TESTS` and having a `DESTRUCTIVE` flag so :+1: